src/components/Input/Input.stories.tsx

```diff
@@ -176,3 +176,52 @@ export const inputAttributes = () => (
         </HelperErrorText>
     </>
 );
+
+export const radioInput = () => {
```

Comment:
If you ever want a `Radio` component in the future, this is pretty much it (after exposing some props).